models/user.php

```diff
@@ -21,4 +30,33 @@ public function register(){
 		}
 		return;
 	}
+	public function login(){
+		// Sanitize POST
+		$post = filter_input_array(INPUT_POST, FILTER_SANITIZE_STRING);
+
+		$password = md5($post['password']);
+
+		if($post['submit']){
+			// Compare Login
+			$this->query('SELECT * FROM users WHERE email = :email AND password =:password');
+			$this->bind(':email', $post['Email']);
+			$this->bind(':password', $password);
+			
+			$row = $this->single();
+
+			if($row){
+				$_SESSION['is_logged_in'] = true;
+				$_SESSION['user_data'] = array(
+					"id" => $row['id'],
+					"name" => $row['name'],
+					"email" => $row['email']
+				);
```

Comment:
use as PHP 7 suggests the brackets for array
```suggestion
				$_SESSION['user_data'] = [
					"id" => $row['id'],
					"name" => $row['name'],
					"email" => $row['email']
				];
```